services/auth/config.yml

```
@@ -115,6 +117,9 @@ defaults:
   # new relic config (processed in services/prelude.js)
   newRelic: !env:json:secret:optional NEW_RELIC
 
+production:
+  app:
+    auditHistoryRetentionDays:  !env AUDIT_RETENTION_DAYS
```

```suggestion
    auditHistoryRetentionDays:  !env:optional AUDIT_RETENTION_DAYS
```

I think it should be optional since we already have a default value for it

---

made the env optional